cmd/manager/main.go

```diff
@@ -65,14 +67,22 @@ func main() {
 	mgr, err := manager.New(cfg, manager.Options{
 		Scheme:             scheme,
 		Namespace:          namespace,
-		MetricsBindAddress: monitoring.GetMetricsAddress(),
```

Comment:
I think this is the only place this function is used, so please also remove the function.